### PR TITLE
Move /resource/ui to /ui

### PR DESCRIPTION
**Pull request by BramGruneir**
This moves ui to a top level directory.

I'm working on adding a package.json and srinkwrap.json file for the next pr.

### CODE REVIEW

**Comment by BramGruneir** [comment]
@tamird and @mrtracy can you take a look please.

File: ui/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## Embedded UI
```
Comment (tamird) [inline ui/README.md]: s/##/#/

File: ui/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+This directory contains the client-side code for cockroach's web admin console.
+These files are embedded into the cockroach binary via the
+[go-bindata](https://github.com/jteeuwen/go-bindata) package, which is used to
+generate the `embedded.go` file in this directory.
+
+#### Modification
```
Comment (tamird) [inline ui/README.md]: s/####/##/g

File: resource/resource.go [hunk 3]
```diff
@@ -15,8 +15,4 @@
 //
 // Author: Matt Tracy (matt@cockroachlabs.com)
```
Comment (tamird) [inline resource/resource.go]: looks to me like this file can die?

**Comment by tamird** [comment]
LGTM but red

**Comment by tamird** [comment]
Oh the red is spurious

**Comment by mrtracy** [comment]
LGTM

File: ui/README.md (hunk 1 above)
Comment (BramGruneir) [inline ui/README.md]: done

File: ui/README.md (hunk 2 above)
Comment (BramGruneir) [inline ui/README.md]: done

File: resource/resource.go (hunk 3 above)
Comment (BramGruneir) [inline resource/resource.go]: It can.  Killed it.

**Pull request merged**